### DISCUSSION TITLE
Task 4_LAB 04.01 - CLEANING DATA.ipynb

### DISCUSSION THREAD

**Discussion by ThomasPineda** [rramosp/ai4eng.v1#182] [04 - GESTIÓN DE DATOS]
Hola, lo que pasa es que hago la tarea 4 como dice en la guia, pero me sale un error que no entiendo. Adjunto evidencia
![image](https://user-images.githubusercontent.com/86691796/184359177-e126b375-d304-423b-85d7-d1a74b1489d3.png)
![image](https://user-images.githubusercontent.com/86691796/184359247-44190cb7-7ec5-4e5a-b8e0-a7edb4f123da.png)
![image](https://user-images.githubusercontent.com/86691796/184359449-e8f931c0-9f25-4593-b2ad-0341a7edaaaf.png)

Mi codigo fue: 
d02 = pd.get_dummies(d01['place'])
d02 = d02.rename({'Bogota': 'place_Bogota', 'Madrid': 'place_Madrid', 'Medellin': 'place_Medellin'}, axis=1)
d01 = d01.drop(['place'], axis=1)
r04 = pd.concat([d02,d01], axis=1)
r04

**Comment by Felipe-RA** [comment]
Saludos ThomasPineda

El error es un `KeyError`, no se encontró una llave en el diccionario, específicamente la llave `place`

**Reply by ThomasPineda** [reply]
Entiendo, ya lo resolví, tuve que cambiar el nombre de la variable en la tercer linea de codigo por "d03" y no por "d01" ya que asi se llama el primer dataset
